projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table-panel/comparison-table-panel.component.html

```diff
@@ -1,7 +1,11 @@
 <ng-container *ngIf="comparisonPanel$ | async as panel">
   <ng-container *ngIf="billingData$ | async as billingInfo">
     <div class="fixed-column">
-      <div class="table-header"></div>
+      <div class="table-header text-center mb-0">
+        <ng-container *ngIf="(user$ | async).uid">
```

Comment:
Although restrictions are handled on the BE, this ensures that no unnecessary calls would be fired to BE for this CMS component if user is not logged in.